http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GceInterpreter.scala

```diff
@@ -132,10 +138,7 @@ class GceInterpreter[F[_]: Parallel: ContextShift: Logger](
       userDisk <- params.runtimeConfig match {
         case x: RuntimeConfig.GceWithPdConfig =>
           for {
-            diskId <- F.fromEither(
-              x.persistentDiskId
-                .toRight(new RuntimeException("Missing diskId in the request. This should never happen"))
-            )
+            diskId <- x.persistentDiskId.get
```

Comment:
why changing this to `get`? the original code is the correct

Reply:
my bad, i changed it back and pushed recently